app/controllers/association/recognition/create.js

```diff
@@ -0,0 +1,3 @@
+import AssociationRecognitionEditController from './edit';
+
+export default class AssociationRecognitionCreateController extends AssociationRecognitionEditController {}
```

Comment:
this is weird, from controllers to extend each other, I would just repeat the code, in the future we might add things to the Edit controller that we don't want in the create